plugins/core/IO/qLASIO/include/LasIOFilter.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+//##########################################################################
+//#                                                                        #
+//#                CLOUDCOMPARE PLUGIN: LAS-IO Plugin                      #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 of the License.               #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#                   COPYRIGHT: Thomas Montaigu                           #
+//#                                                                        #
+//##########################################################################
+
+#include "FileIOFilter.h"
+
+class LasIOFilter : public FileIOFilter
+{
+  public:
+    LasIOFilter();
+
+    // Inherited from FileIOFilter
+    CC_FILE_ERROR
```

Comment:
the limit of 120 characters in the clang format file may not help to make the code more readable?